src/command/Commands.js

```diff
@@ -110,8 +110,6 @@ define(function (require, exports, module) {
     exports.TOGGLE_LINE_NUMBERS         = "view.toggleLineNumbers";     // EditorOptionHandlers.js      _getToggler()
     exports.TOGGLE_ACTIVE_LINE          = "view.toggleActiveLine";      // EditorOptionHandlers.js      _getToggler()
     exports.TOGGLE_WORD_WRAP            = "view.toggleWordWrap";        // EditorOptionHandlers.js      _getToggler()
-    exports.TOGGLE_ALLOW_JAVASCRIPT     = "cmd.toggleAllowJavaScript";  // EditorOptionsHandlers.js     _getToggler()
-    exports.TOGGLE_AUTO_UPDATE          = "cmd.toggleAutoUpdate";       // EditorOptionsHandlers.js     _getToggler()
```

Comment:
You need to restore this file to how it should be.  Use `git checkout master src/command/Commands.js` to fix.